numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -4744,6 +4744,8 @@ PyMODINIT_FUNC init_multiarray_umath(void) {
     if (initumath(m) != 0) {
         goto err;
     }
+    /* Initialize CPU features */
+    npy_cpu_init();
```

Comment:
`npy_cpu_have` requires to call `npy_cpu_init` first, Should we add it on top? also is there other uncovered areas?

Reply:
`npy_cpu_init` must be called before `InitOperators`

Reply:
hm one could just `cpu_init` set a static variable and have `cpu_have` call it when needed